topbeat/beater/config.go

```diff
@@ -1,17 +1,48 @@
 package beater
 
+import (
+	"errors"
+	"time"
+)
+
+type ConfigSettings struct {
```

Comment:
I didn't mean to do it fully automatically, just that we put the section into the unpack for each beat. But if we can fully automate it in the future, even better.